[16.12.2024 14:01:03,64] Running andel_flyktninger_og_arbeidsinnvandrere.py 
D:\Scripts\analyse\Telemark\Python\Queries\09_Innvandrere_og_inkludering\Innvandrerbefolkningen\andel_flyktninger_og_arbeidsinnvandrere.py:153: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  df_pivot_arbeidsinnvandrere["Label"] = df_pivot_arbeidsinnvandrere["Kommune"]
D:\Scripts\analyse\Telemark\Python\Queries\09_Innvandrere_og_inkludering\Innvandrerbefolkningen\andel_flyktninger_og_arbeidsinnvandrere.py:154: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  df_pivot_flyktninger["Label"] = df_pivot_flyktninger["Kommune"]
Loaded .env file from: D:\Scripts\analyse\Telemark\Python\token.env
X_FUNCTIONS_KEY loaded successfully.
Loaded .env file from: D:\Scripts\analyse\Telemark\Python\token.env
GITHUB_TOKEN loaded successfully.
Flyktninger og arbeidsinnvandrere CSV data loaded successfully.
Selected data from year 2024-01-01 00:00:00

Debug information:
Numeric columns detected: ['andel arbeidsinnvandrere 2024']
Key columns detected: ['kommune', 'label']

Sample of new data types:
kommune                          float64
andel arbeidsinnvandrere 2024      int64
label                            float64
dtype: object

Sample of existing data types:
kommune                          float64
andel arbeidsinnvandrere 2024      int64
label                            float64
dtype: object
New data detected. Uploading to GitHub.

Columns in merged comparison:
['kommune', 'andel arbeidsinnvandrere 2024_old', 'label', 'andel arbeidsinnvandrere 2024_new']

=== Data Comparison ===
File: andel_arbeidsinnvandrere.csv
Changes detected in the following rows:

Row identifiers:
  kommune: nan
  label: nan
Value changes:
Traceback (most recent call last):
  File "D:\Scripts\analyse\Telemark\Python\Queries\09_Innvandrere_og_inkludering\Innvandrerbefolkningen\andel_flyktninger_og_arbeidsinnvandrere.py", line 182, in <module>
    is_new_data1 = compare_to_github(
  File "D:\Scripts\analyse\Telemark\Python\Helper_scripts\github_functions.py", line 250, in compare_to_github
    print(f"  {col}: {row[old_col]} \u2192 {row[new_col]}")
  File "C:\Users\_eve1509\AppData\Local\anaconda3\envs\analyse\lib\encodings\cp1252.py", line 19, in encode
    return codecs.charmap_encode(input,self.errors,encoding_table)[0]
UnicodeEncodeError: 'charmap' codec can't encode character '\u2192' in position 35: character maps to <undefined>
[16.12.2024 14:01:07,24] Script failed with error code 1 
